web/src/components/home-page/intro-video.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Box, Container, SystemStyleObject } from 'theme-ui';
+import YouTubeVideo from './youtube-video';
+
+interface IntroVideoProps {
+  videoUrl?: string;
+  videoTitle?: string;
+}
+
+const style: SystemStyleObject = {
+  '.video-container': {
+    position: 'absolute',
+    top: 0,
+    bottom: 0,
+    right: 0,
+    left: 0,
+    zIndex: -99,
+    '&::after': {
+      display: 'block',
+      content: '',
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '800px',
```

Comment:
We should probably use dynamic values. Consider replacing `height` with `maxHeight` or `minHeight`.

Reply:
You could also use the size matrix if it should be a fixed value.